src/main/java/com/minecolonies/coremod/colony/Colony.java

```diff
@@ -586,7 +596,7 @@ public void updateSubscribers()
 
                     final double distance = player.getDistanceSq(center);
                     if (distance < MathUtils.square(Configurations.workingRangeTownHall + 16D)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Assign this magic number 16D to a well-named constant, and use the constant instead. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS109)